WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2054,3 +2054,17 @@ ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
 MetaPropertyTable       =
 MetaPropertyTableCell   = padding:@SkinWrapperPadding@ @WidgetInternalPadding@; padding-top:0;
 
+#######################################################################
+#   Task List view
+#######################################################################
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right: 12px;
+TaskListViewNewTaskBannerWrapper                            = @Text-hint@ @InactiveCursor@ @FontSize-big@ padding:16px 0 16px 0; margin-left:6px;
```

Comment:
The same comment which i made earlier :

margin-left is pushing the dynamically created input field to the left by 6px. padding-left should be used here.
ZmTaskListView.prototype._resetInputSize() function calculating the input position based on the div position which should not be changed . (Please have a look for reference: https://screencast.com/t/9YXKYxpW51Fr )